tests/test_create.py

```diff
@@ -2589,3 +2589,15 @@ def test_repodata_v2_base_url(
         platform,
     )
     assert package_is_installed(prefix, "ca-certificates")
+
```

Comment:
Could we please add a new test case to test `conda create python` (i.e. no `--name` or `--prefix` provided), so we can cover the new `raise ArgumentError` exception you added in `main_create.py`?

Reply:
Done :)